### ISSUE TITLE
Javascript error when trying to view pipeline created based on template

### ISSUE THREAD

**Issue by costimuraru** [spinnaker/spinnaker#4059]
### Issue Summary:

We created a pipeline-template using `spin`. 
We then created a pipeline based on this template, also by using `spin`.
However, when trying to view the pipeline from the UI, we are receiving a javascript error and the page doesn't load.

```
exports.default @ vendors~app.js:41
vendors~app.js:278 TypeError: Cannot read property 'pipeline' of undefined
    at initialize (app.js:108542)
    at vendors~app.js:278
    at vendors~app.js:278
    at Scope.$digest (vendors~app.js:278)
    at Scope.$apply (vendors~app.js:278)
    at vendors~app.js:278
    at completeOutstandingRequest (vendors~app.js:278)
    at vendors~app.js:278 "Possibly unhandled rejection: {}"
```

<img width="1073" alt="screenshot 2019-03-01 16 01 07" src="https://user-images.githubusercontent.com/952836/53643160-09b3e680-3c3c-11e9-9f8f-7a6fb624020d.png">
<img width="1407" alt="screenshot 2019-03-01 16 01 03" src="https://user-images.githubusercontent.com/952836/53643164-0b7daa00-3c3c-11e9-86c1-deb4221420cd.png">

<img width="1431" alt="screenshot 2019-03-01 16 02 30" src="https://user-images.githubusercontent.com/952836/53643152-03be0580-3c3c-11e9-95c3-a258adf672c3.png">

### Environment:
We're using Spinnaker `1.12.2` deployed in Kubernetes.

### Feature Area (if this issue is UI/UX related, please tag @spinnaker/ui-ux-team):
UI

### Steps to Reproduce:

1) spin pipeline-template save -f pipeline-template.json
2) spin pipeline save -f pipeline.json 
3) Go in the UI, and try to view the pipeline.

`pipeline-template.json`:
```json
 {
  "schema" : "v2",
  "variables" : [
  {
    "type" : "int",
    "defaultValue" : 42,
    "description" : "The time a wait stage shall pauseth",
    "name" : "waitTime"
  }
  ],
  "id" : "pipescu",
  "protect" : false,
  "metadata" : {
    "name" : "Variable Wait",
    "description" : "A demonstrative Wait Pipeline.",
    "owner" : "example@example.com",
    "scopes" : ["global"]
  },
  "pipeline": {
    "lastModifiedBy" : "anonymous",
    "updateTs" : "0",
    "parameterConfig" : [],
    "limitConcurrent": true,
    "keepWaitingPipelines": false,
    "description" : "",
    "triggers" : [],
    "notifications" : [],
    "stages" : [
    {
      "expectedArtifacts": [
        {
          "defaultArtifact": {},
          "matchArtifact": {
            "name": "kuber-app",
            "type": "embedded/base64"
          },
          "useDefaultArtifact": false
        }
      ],
      "inputArtifacts": [
        {
          "account": "helm-aam-test-snapshot"
        }
      ],
      "name": "Bake (Manifest)",
      "namespace": "kubernetes-demo-app",
      "outputName": "kuber-app",
      "overrides": {
        "image.tag": "portocale"
      },
      "requisiteStageRefIds": [],
      "templateRenderer": "HELM2",
      "type": "bakeManifest"
    }
    ]
  }
}
```

`pipeline.json`:
```json
 {
  "schema": "v2",
  "application": "testkubev2",
  "name": "Fresh new MPT v2 pipeline",
  "template": {
    "source": "spinnaker://pipescu"
  },
  "variables": {
    "waitTime": 4
  },
  "inherit": [],
  "triggers": [],
  "parameters": [],
  "notifications": [],
  "description": "",
  "stages": []
}
```

**Comment by DavidGereb** [comment]
the UI doesn't yet support v2
I think it's related to #3930

**Comment by jtk54** [comment]
@louisjimenez

**Comment by louisjimenez** [comment]
Thanks for the detailed info, this is being covered as part of https://github.com/spinnaker/spinnaker/issues/4056

**Comment by louisjimenez** [comment]
https://github.com/spinnaker/spinnaker/issues/4056 removed configure links and setup a redirect when viewing MPT v2 pipelines. It is patched in 1.12.5. https://github.com/spinnaker/spinnaker/issues/4116 will add support for the MPT v2 pipelines configuration view.
